site/content/contribute/server/developer-setup/centos.md

```diff
@@ -58,17 +58,23 @@
     {username}  hard  nofile  8096
     ```
 
-6. Log out and log back in to effect the changes above.
+6. Add your user to the `docker` group, replacing `{username}` with your username:
 
-7. Fork https://github.com/mattermost/mattermost-server.
+    ```sh
+    sudo usermod -aG docker {username}
```

Comment:
Bullet point 1. includes `    sudo usermod -aG docker $(whoami)`. Does this already cover this issue?

Reply:
Oh my, so it does. My mistake. Sorry for the noise.